packages/patterns-react/src/patterns/LogoGrid/__stories__/LogoGrid.stories.js

```diff
@@ -0,0 +1,82 @@
+import './index.scss';
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import { DDS_LOGO_GRID } from '../../../internal/FeatureFlags';
+import LogoGrid from '../LogoGrid';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+const logosGroup = [
+  {
+    label: 'PHILIPS',
```

Comment:
Change the company names here as well, probably shouldn't have actual company names in our code.